app/views/govuk_publishing_components/component_guide/preview.html.erb

```diff
@@ -0,0 +1,8 @@
+<% @component_fixtures.each do |fixture| %>
+  <div class="component-guide-preview-page">
+    <% unless @component_fixtures.length <= 1 %>
```

Comment:
I'd switch this to be a positive check, eg:
`if @component_fixtures.length > 1`